README.md

```diff
@@ -1,8 +1,8 @@
 # ES Module Attributes
 
-Champions: Sven Sauleau ([@xtuc](https://github.com/xtuc)) and Daniel Ehrenberg ([@littledan](https://github.com/littledan))
+Champions: Sven Sauleau ([@xtuc](https://github.com/xtuc)), Daniel Ehrenberg ([@littledan](https://github.com/littledan)), and Myles Borins ([@MylesBorins](https://github.com/MylesBorins))
```

Comment:
do we want to add Daniel Clark here too? Not sure github handle

Reply:
cc @dandclark How do you want to be presented here?

Reply:
Happy to co-champion if you'd like me on board.  Created a [PR](https://github.com/tc39-transfer/proposal-module-attributes/pull/32).